lib/thredded_create_app/tasks/docker/docker-compose.yml.erb

```diff
@@ -1,4 +1,4 @@
-version: "3.6"
+version: "3.7"
```

Comment:
Is this necessary? The version of docker-compose that comes with Ubuntu 19.04 doesn't support 3.7

Reply:
Ah good call, will revert this :+1: